docs/included-checks.md

```diff
@@ -0,0 +1,58 @@
+# ESH Guidelines Covered

```

Comment:
I have found more problems with the existing approach - the link actually points to a row in the ruleset.xml file, and these rows are not up to date after several checks merged :(. I like the idea of several columns, so maybe three columns will be needed (the rule, the name of the check and the severity).

Adding specific colors might be difficult, I will see if I can attach some include some CSS for that.